src/types.ts

```diff
@@ -14,6 +14,7 @@ export interface Options {
   root?: string;
   pack?: PackType;
   manifest?: string[];
+  attw?: boolean;
```

Comment:
this is what i was trying to explain in discord the other day: we shouldn't be exposing any of this as "attw".

we should have a feature to enable "type checking". right now we happen to use attw under the hood, but we may not in future. it shouldn't matter to the user